plutus-pab-executables/tx-inject/config.yaml

```diff
@@ -35,7 +36,7 @@ nodeServerConfig:
     - getWallet: 8
     - getWallet: 9
     - getWallet: 10
-  mscNodeMode: AlonzoNode
+  pscNodeMode: AlonzoNode
```

Comment:
Since you modified this config file, you could do the same with `plutus-pab-executables/plutus-pab.yaml.sample`?

Reply:
Done.